Frontend/android-native/app/src/user/nammaYatri/assets/juspay/juspay_assets.json

```
@@ -605,6 +608,7 @@
       "ny_ic_profile_shine" : true,
       "ny_ic_prohibited" : true,
       "ny_ic_qr_code" : true,
+      "ny_ic_qr_code_expired" : true,
```

Biome (1.9.4)

[error] The key ny_ic_qr_code_expired was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)